apps/android_camera/app/src/main/res/drawable/item_selector.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+ Copyright 2014 The Android Open Source Project
```

Comment:
GIven these are files from Android Open Source Project instead of the the ASF, we will need to list these files as in https://github.com/apache/incubator-tvm/blob/master/LICENSE#L215

Reply:
And add LICENSE.aosp.txt https://github.com/apache/incubator-tvm/tree/master/licenses